clickfile.py

```diff
@@ -75,6 +75,7 @@
 HOODIES_CHAINLINK_GITHUB_URL = "https://github.com/hoodieshq/chainlink-neon"
 PROXY_GITHUB_URL = f"https://api.github.com/repos/{DOCKER_HUB_ORG_NAME}/neon-proxy.py"
 FAUCET_GITHUB_URL = f"https://api.github.com/repos/{DOCKER_HUB_ORG_NAME}/neon-faucet"
+UNISWAP_V3_GIT = f"git@github.com:{DOCKER_HUB_ORG_NAME}/Uniswap-V3-NEON.git"
```

Comment:
lets make it as for others parts by http (for ssh there are should be ssh github configuration)

Reply:
I did it because I got 403 from http, I'll ask devopses to help and then fix it

Reply:
fixed